website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Scorecard (alpha)
+weight: 25
+---
+
+# operator-sdk scorecard (alpha)
+
+## Overview
+
+The scorecard command, part of the operator-sdk, executes tests 
+on your operator based upon a user defined configuration file 
+and test images.
+
+Tests are implemented within test images that are configured 
+and constructed to be executed by scorecard.
+
+Each test(s) is executed within a Pod by scorecard, pod logs 
+contain the test results, and scorecard aggregates the test 
+results to display to end users.  Scorecard therefore assumes 
+it is being executed with access to a configured Kube cluster.
+
+Scorecard has built-in basic and OLM tests but also provides a 
+means to execute custom user defined tests.
+
+## Requirements
+
+The scorecard tests make no assumptions as to the state of the 
+operator being tested.  Creating operators and custom resources 
+for an operator are left outside the scope of the scorecard itself.
+
+Scorecard tests can however create whatever resources they 
+require if the tests are designed for resource creation.
+
+Scorecard requires the following Kube environment:
+
+- Access to a Kubernetes v1.17.1+ cluster
```

Comment:
Why we need v1.17.1+ cluster? Should not it be the same pre-requirement of SDK ?

Reply:
good catch, I updated it to match the 1.11.3 specified in the current scorecard doc.